### PR TITLE
adding onboarding details to proposal page for Neighborhood Assistance 

### PR DESCRIPTION
**Pull request by jenirainerpdx**

### CODE REVIEW

**Comment by jenirainerpdx** [comment]
Please add Pedro Filipe to requested reviewers.  Thank you!

File: software/proposals/neighbourhood_assistance.md [hunk 1]
```diff
@@ -53,6 +55,25 @@ Software to help people organize neighborhood assistance.
...
+https://github.com/puzzledbytheweb/neighborhood-assistance
+
+### Frontend / UI repo: 
+https://github.com/comoser/neighborhood-assistance-web
+
+### Contributors
+https://docs.google.com/spreadsheets/d/1HOhIguRcVHtiFq7fGSgdDP_XMhWLiDh_PsFm0X7U_8c/edit#gid=0
+
+### Technical Leads (slack names)
+@Pedro Filipe
```
Comment (comoser) [inline software/proposals/neighbourhood_assistance.md:74]: not only Pedro on this but also Shayan Chowdhury

**Comment by s0** [comment]
This is great!

We should probably also remove the link to the old / outdated Google doc, see: https://trello.com/c/wHAa6uNw/24-clean-up-project-info-docs-spread-around-the-place

File: software/proposals/neighbourhood_assistance.md [hunk 2]
```diff
@@ -53,6 +55,25 @@ Software to help people organize neighborhood assistance.
...
+
```
Comment (jenirainerpdx) [inline software/proposals/neighbourhood_assistance.md]:
```suggestion
@Shayan Chowdhury
```

**Comment by jenirainerpdx** [comment]
> This is great!
> 
> We should probably also remove the link to the old / outdated Google doc, see: https://trello.com/c/wHAa6uNw/24-clean-up-project-info-docs-spread-around-the-place

I am not clear on what has been deprecated re: google docs.

**Review by puzzledbytheweb — APPROVED** [review]
Looks good to me!

Thanks!

**Review by mazaira — APPROVED** [review]
Thanks for keeping it updated :)

**Pull request merged**